src/main/java/com/onarandombox/MultiverseCore/display/parsers/ContentParser.java

```diff
@@ -0,0 +1,20 @@
+package com.onarandombox.MultiverseCore.display.parsers;
+
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+/**
+ * Parse objects into string or list of strings.
+ */
+@FunctionalInterface
+public interface ContentParser {
+    /**
+     * Parse the object to string(s) and add it to the content.
+     *
+     * @param sender    The target which the content will be displayed to.
+     * @param content   The content to display.
+     */
+    void parse(@NotNull CommandSender sender, @NotNull List<String> content);
```

Comment:
I think this might be better if it returns the content list instead of is given one to add to. I think this is just a gut feeling and maybe isn't entirely right but I feel like a list you're given to modify smells a bit more than just returning a list.

Reply:
The reason why this doesn't just return a list of strings so we only need one common list. Instead of each parsing creating a new list as then adding it to another common list... which is potentially a lot of lists. 

For whether if we delete/modify existing entries from other parsers, either we allow it as a feature, or just instead pass a wrapper around the List<String> that only allow adding of elements.

But all these won't be needed if we decide to only allow one content parser.